### PR TITLE
The packaging email as a blog post.

### PR DESCRIPTION
**Pull request by pilhuhn**

### CODE REVIEW

File: src/main/jbake/content/blog/2016/04/28/new-packaging.adoc [hunk 1]
```diff
@@ -0,0 +1,129 @@
+= New Hawkular packaging
+Heiko W. Rupp, Stefan Negrea
+2016-4-28
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, manageiq, components
+
+_ [ This blog post is was sent before to the hawkular-dev mailing list ] _
```
Comment (jkremser) [inline src/main/jbake/content/blog/2016/04/28/new-packaging.adoc]: ...post is was sent...  + it looks strange on the page http://209.132.178.114:10167/blog/2016/04/28/new-packaging.html  Not sure what the `_ [ something ] _` should do in adoc, but it seems not to work for the website

File: src/main/jbake/content/blog/2016/04/28/new-packaging.adoc [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+=== Hawkular Core Services
+
+Geared towards ManageIQ integrations and will include only the components needed for the MiQ provider.
+This package uses the Metrics Distribution as described below.
+
+Components included: Metrics, Alerts, Inventory, CommandGW, Nest and Bus as well as the agent.
+
+Reasons for some exclusions: Keycloak (different security model), Embedded Cassandra
+(only full C* deployments will be supported), and no Hawkular UI because it will use a special purpose UI for ManageIQ
```
Comment (jkremser) [inline src/main/jbake/content/blog/2016/04/28/new-packaging.adoc:90]: ~~`)` is missing~~

File: src/main/jbake/content/blog/2016/04/28/new-packaging.adoc (hunk 1 above)
Comment (abonas) [inline src/main/jbake/content/blog/2016/04/28/new-packaging.adoc]: "is was" sounds strange. Perhaps "has been sent" is better

File: src/main/jbake/content/blog/2016/04/28/new-packaging.adoc [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+With a growing community and an increasingly large number of sub-projects, it is the perfect time to rethink the
+packaging of Hawkular. This post is a blueprint for changes to come to Hawkular as well as establish some patterns
+for futures decisions.
+
+== Motivation
+
+So far we have created Hawkular "All-in-one" releases that contain everything, Hawkular server components, UI,
+the agent and also embedded versions of Cassandra and KeyCloak.
+
```
Comment (abonas) [inline src/main/jbake/content/blog/2016/04/28/new-packaging.adoc:18]: "It’s certainly the case that for community we will still deliver a fully featured all-in-one package" - is this a valid statement given the shift of the UI to miq? cc @pilhuhn @theute

File: src/main/jbake/content/blog/2016/04/28/new-packaging.adoc (hunk 1 above)
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/04/28/new-packaging.adoc]: Yeah, just wanted to check if anyone pays attention :-)

**Pull request merged**